db/schema.rb

```diff
@@ -74,6 +74,14 @@
     t.index ["reset_password_token"], name: "index_companies_on_reset_password_token", unique: true
   end
 
+  create_table "companies_roles", id: false, force: :cascade do |t|
```

Comment:
this table should not exist @acolletti21 There might be some stale migration files in your repo